libraries/ms-drivers/src/x86/pca9539r_gpio_expander.c

```diff
@@ -81,3 +81,14 @@ StatusCode pca9539r_gpio_get_state(const Pca9539rGpioAddress *address,
   *input_state = s_pin_settings[address->i2c_address][address->pin].state;
   return STATUS_CODE_OK;
 }
+
+StatusCode pca9539r_gpio_subscribe_interrupts(const GpioAddress *interrupt_pin,
+                                              const Pca9539rInterruptCallback callback,
+                                              void *context) {
+  gpio_it_init();
```

Comment:
Libraries like `gpio_it` can generally only be initialized once per program, or else they do nasty things like resetting all the registered interrupts. Because of this, we leave initializations to the `main()` function in each project, and we tell projects to initialize the relevant libraries using a comment at the top of the header file (like `// Requires ... to be initialized.`).

So remove this line, and add a line at the top of the header underneath the `// Requires I2C to be initialized.` line that says something about how if you want to subscribe to interrupts, you must initialize `gpio_it`.